app/models/candidate_interface/pick_course_form.rb

```diff
@@ -7,12 +7,12 @@ class PickCourseForm
     validate :user_cant_apply_to_same_course_twice
 
     def applyable?
-      course.open_on_apply?
+      course.open_on_apply? && course.exposed_in_find?
```

Comment:
This duplicates the logic from the `open_on_apply` scope — is there a way to dry it up?

Reply:
Nope, they're boolean attributes. I don't see any way to combine those?